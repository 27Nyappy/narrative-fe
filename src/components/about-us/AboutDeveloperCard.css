.AboutDeveloperCard {
  background-color: yellow;
  width: 30%;

  div {
    background-color: palegreen;
    display: flex;
    flex-direction: column;
    justify-content: center;
    align-items: center;
    padding: 50px;
  }
}