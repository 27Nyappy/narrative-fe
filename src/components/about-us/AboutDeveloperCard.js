import React from 'react';
import PropTypes from 'prop-types';

const AboutDeveloperCard = ({ developerName, developerImage, developerBio }) => {
  return (
    <>
      <h2>{developerName}</h2>
      <div>
        <img src={developerImage} alt={developerName} />
        <p>{developerBio}</p>
      </div>
    </>
  );
};

AboutDeveloperCard.propTypes = {
  developerName: PropTypes.string.isRequired,
  developerImage: PropTypes.string.isRequired,
  developerBio: PropTypes.string.isRequired
};

export default AboutDeveloperCard;