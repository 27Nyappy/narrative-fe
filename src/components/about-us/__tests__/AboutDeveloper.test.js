import React from 'react';
import { shallow } from 'enzyme';
import AboutDeveloperCard from '../AboutDeveloperCard';

describe('AboutDeveloperCard component', () => {
  it('renders an about developer card', () => {
    const wrapper = shallow(<AboutDeveloperCard developerName="developer" developerImage="developer.jpg" developerBio="I am a developer"/>);
    expect(wrapper).toMatchSnapshot();
  });
});