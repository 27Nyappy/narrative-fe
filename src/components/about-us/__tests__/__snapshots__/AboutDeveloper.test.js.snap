// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutDeveloper component renders an about developer card 1`] = `
<Fragment>
  <h2>
    developer
  </h2>
  <div>
    <img
      alt="developer"
      src="developer.jpg"
    />
    <p>
      I am a developer
    </p>
  </div>
</Fragment>
`;
